crochetcraft/src/routes/demos/chain/+page.svelte

```diff
@@ -20,19 +22,7 @@
 
     // Chain stitch as three cubic Beziers
     // Fairly innaccurate but has some of the geometry, at least
-    const ChainStitchParts = makeMultiBezier([
-        new Vector3(0.0, 0.0, 0.0),
-        new Vector3(0.03989, -0.1529, 0.4265),
-        new Vector3(-0.1251, 0.1748, 0.4262),
-        new Vector3(-0.286, 0.4362, -0.05045),
-        new Vector3(-0.4751, 0.7433, -0.6106),
-        new Vector3(-0.8814, 1.054, 0.3796),
-        new Vector3(-0.1089, 1.246, 0.1362),
-        new Vector3(0.6817, 1.442, -0.1128),
-        new Vector3(-0.4605, 0.585, -0.5125),
-        new Vector3(-0.01001, 0.4943, -0.007016),
-    ]);
-
+    const ChainStitchParts = makeMultiBezier(arrayToVector3List(models.chainStitch.points));
```

Comment:
I wonder if the `arrayToVector3List` should be done in the crochet-stitcher library, so that it exports the `Vector3[]` instead of the raw JSON? That way, we don't have to convert it every time we use it. And it would be nice to have a proper type for it.

Reply:
Vector3 is part of THREE, so we'd have to add it to crochet-stitcher, which feels a little weird since it should just be used for frontend rendering

either way I didn't want to change more dependencies than needed

Reply:
That being said, converting the json to a ts is reasonable once we have a better idea of what data we need

Reply:
Hmm, okay, I suppose that's fair. In that case, could we add type definitions closer to the JSON file? Something like

```ts
export interface StitchModel {
    curveType: 'bezier';
    points: [number, number, number][];
}

export const CHAIN: StitchModel = chainStitch;
```

Reply:
Added it, no idea if my type import worked or if imports are just borked in general, but I don't really want this PR to be about fixing imports when it's just supposed to move out the chain stitch from the one place it was being used in